contracts/programs/ocr2/src/lib.rs

```diff
@@ -783,30 +789,44 @@ impl Report {
     }
 }
 
-fn calculate_reimbursement(juels_per_lamport: u64, _signature_count: usize) -> Result<u64> {
-    const SIGNERS: u64 = 1;
+fn calculate_reimbursement_gjuels(juels_per_lamport: u64, _signature_count: usize) -> Result<u64> {
+    const SIGNERS: u128 = 1;
+    const GIGA: u128 = 10u128.pow(9);
     let fees = Fees::get()?;
-    let lamports_per_signature = fees.fee_calculator.lamports_per_signature;
+    let lamports_per_signature: u128 = fees.fee_calculator.lamports_per_signature.into();
     let lamports = lamports_per_signature * SIGNERS;
-    let juels = lamports * juels_per_lamport;
-    Ok(juels)
+    let juels = lamports * u128::from(juels_per_lamport);
```

Comment:
Actually, `lamports_per_signature` is guaranteed to be a small value (`1 * 5000`) so you could instead convert it here:

```rust
let juels = u128::from(lamports) * u128::from(juels_per_lamport);
```